UnitySDK/Assets/ML-Agents/Scripts/Sensor/ISensor.cs

```diff
@@ -56,4 +56,24 @@ public interface ISensor {
         string GetName();
     }
 
+    public static class ISensorExtensions
```

Comment:
I guess this is the equivalent of adding code in a header file.  Nice.

Reply:
Yeah, I just wanted an implementation that used the interface method. https://stackoverflow.com/questions/2770333/can-extension-methods-be-applied-to-interfaces says it's OK.